docs/design/launch_system.md

```diff
@@ -22,15 +23,21 @@ Each folder contains a `launch/` folder where all the launch files should be pla
 
 ### File Naming Convention
 
-All launch files end in `.launch.py`. Furthermore, all launch files specific to the a vehicle platform or orchastrator launch files are prefixed with `art_`.
+All launch files end in `.launch.py`. Furthermore, all launch files specific to a vehicle platform or orchastrator launch files are prefixed with `art_`.
 
 ## `art_launch/`
 
-This is where the main launch file is held: [`art.launch.py`](../../workspace/src/common/launch/art_launch/launch/art.launch.py). This file will do a few things.
+This is where the main launch file is held: [`art.launch.py`](../../workspace/src/common/launch/art_launch/launch/art.launch.py). The only purpose of this file is to define system wide parameters (e.g. `LaunchConfigurations`, `LaunchDescriptions`, etc.). This will allow all orchestrators to know which specific packages to launch, and which to ignore. Following this, ther remainder of the general setup is left to the `art_shared_launch`.
```

Comment:
Following this, **the**...

But also not sure about the idea generally. Is this what we agreed on? Why have art_shared_launch at all? Why not put the stuff in art_shared_launch in art_launch?

Reply:
Wasn't that what you were asking me to do in #113? or did I miss interpret that?

Reply:
Ya sorry, I guess I think `art_launch` should be that common launch file, not a new package. Does that make sense to you? Or is there a clear distinction between these two packages?

Reply:
Ok I think I see what you mean. In my mind `art_launch` was more of a "sandbox" or "catch-all" that runs everything, or runs some basic demo. Not everything that is currently in `art_launch` is actually common across launch configurations (e.g. currently disables all localization nodes other than one. What if I want a different one?).  Because of that, I don't like having the `disable` launch arguments in the common file. Does splitting this up into two packages make sense?

I don't care too much about naming conventions. We can make `art_launch` be what is in the `art_shared_launch` atm, and then move what is in `art_launch` to something named something else (`art_sandbox_launch`, art_demo_launch`, something else?). 

Thoughts? Is there a good way / good reason to keep these things coupled?
(p.s. my thought process was that everyone is used to launching using `art_launch`, so keeping this working / unchanged seemed simpler to me. But again, I don't have a preference for naming conventions)

Reply:
Oh okay I'd actually argue [this](https://github.com/uwsbel/autonomy-research-testbed/blob/d4dcb682e8156905f2895f5bbda18c8383794036/workspace/src/common/launch/art_launch/launch/art.launch.py#L81) shouldn't go here haha, ig i missed it in the review. The common launch file should only setup launch configurations and ig probs composability. But disable calls should probs go in individual package files, ya.

So I guess this makes sense to me:
- `art_launch`: For common launch configurations
- `art_dev_launch`: Launch file for the `dev` service. Probs similar to `art_launch` now where it only really is made for simulation.
- `art_<vehicle>_launch`: what you're doing for `oak`.

Reply:
Sweet sounds good, just fixed this. Going to fix your comment (and the described structure) in #113 rn.